tck/README.md

```diff
@@ -436,19 +425,19 @@ public class MySubscriberTest extends BlackboxSubscriberVerification<Integer> {
 }
 ```
 
-Note that hard-coded values *take precedence* over environment set values (!).
+Note: hard-coded values *take precedence* over environment set values (!).
```

Comment:
We had "ATTENTION", should this be "NOTE:" then? 
Some renderers actually use "NOTE:" as floating box with note marker (asciidoctor AFAIR)